### PR TITLE
[FIX]: Return rejected streamlines with empty sft

### PR DESCRIPTION
**Pull request by levje**
# Quick description

As explained here: https://github.com/scilus/scilpy/issues/1073

## Type of change

Check the relevant options.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Provide data, screenshots, command line to test (if relevant)
`pytest scilpy/tractograms/tests/test_streamline_operations.py -k 'test_filter_streamlines_by_length'`

# Checklist

- [x] My code follows the style guidelines of this project (run [autopep8](https://pypi.org/project/autopep8/))
- [x] I added relevant citations to scripts, modules and functions docstrings and descriptions
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I moved all functions from the script file (except the argparser and main) to scilpy modules
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by frheault** [comment]
Is this the same problem or related to issue #1007 ? Can you look into it?

**Review by arnaudbore — COMMENTED** [review]
quick comment

File: scilpy/tractograms/streamline_operations.py [hunk 1]
```diff
@@ -408,19 +408,15 @@ def filter_streamlines_by_length(sft, min_length=0., max_length=np.inf,
...
+        valid_length_ids = np.array([], dtype=bool)
         filtered_sft = sft
 
     # Return to original space
-    sft.to_space(orig_space)
     filtered_sft.to_space(orig_space)
 
     if return_rejected:
-        rejected_sft.to_space(orig_space)
+        rejected_sft = sft[~valid_length_ids]
```
[the remark this answers is not in the document]
Comment (levje) [reply scilpy/tractograms/streamline_operations.py:420]: Just didn't mean to delete `sft.to_space(orig_space)`. Should be good now.

**Review by arnaudbore — APPROVED** [review]
GTG

**Pull request merged**